WooCommerce/WooCommerceTests/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrderBottomSheetListSelectorCommandTests.swift

```diff
@@ -4,16 +4,25 @@ import XCTest
 
 final class ProductsSortOrderBottomSheetListSelectorCommandTests: XCTestCase {
     func testInitialSelectedValue() {
+        // Arrange
         let selected = ProductsSortOrder.nameAscending
-        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: selected)
+        var selectedActions = [ProductsSortOrder]()
+        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: selected) { (selected) in
+            selectedActions.append(selected)
+        }
+        // Action
+        command.handleSelectedChange(selected: .nameAscending)
+        // Assert
         XCTAssertEqual(command.selected, selected)
     }
 
     func testSelectedValueAfterChange() {
```

Comment:
It looks like the original code is testing that `command.selected` changes whenever `command.handleSelectedChange` is called with a different value. Since `handleSelectedChange`'s behavior changed, this is no longer valid. 

You're right that we can replace this with a unit test that verifies that the callback was called. We can replace this test function with something like this: 

```swift
    func test_handleSelectedChange_notifies_the_callback_the_new_selected_value() {
        // Arrange
        var receivedSelectedSortOrder: ProductsSortOrder?
        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: .nameAscending) { (selected) in
            receivedSelectedSortOrder = selected
        }

        // Action
        let newSelected = ProductsSortOrder.dateDescending
        command.handleSelectedChange(selected: newSelected)

        // Assert
        XCTAssertEqual(receivedSelectedSortOrder, newSelected)
    }
```

We can have a local variable, `receivedSelectedSortOrder`, that is updated whenever the callback is called. And test that it's updated to what we expect.